test/src/audits/performance/user-timing.js

```diff
@@ -18,23 +18,17 @@
 const Audit = require('../../../../src/audits/performance/user-timings.js');
 const assert = require('assert');
 const traceContents = require('./trace-user-timings.json');
-
 /* eslint-env mocha */
+
 describe('Performance: user-timings audit', () => {
   it('fails gracefully', () => {
     const output = Audit.audit({});
     assert.equal(output.value, -1);
   });
 
-  it('processes a trace file for user timing data', () => {
+  it('evaluates valid input correctly', () => {
     const output = Audit.audit({traceContents});
-    assert.equal(output.value, 1);
-    assert.equal(output.extendedInfo.value[0].name, 'elapsed');
-  });
-
-  it('handles valid traces with no user timing data', () => {
-    const output = Audit.audit({traceContents: traceContents.slice(0, 1)});
-    assert.equal(output.value, 0);
-    assert.equal(output.extendedInfo.value.length, 0);
+    console.log(output);
+    assert.equal(output.value, 2);
```

Comment:
right now this is missing the `navigationStartTime` assignment in code coverage, which implies that the user timings coming out of the test would be `NaN` (since they'll have `navigationStartTime` subtracted from them, though I haven't run it to verify). Probably want the test to verify the `extendedInfo` output, which is a good thing anyway as we better guarantee our `--output=json` output so CI systems can depend on the format